src/stream/zstd.rs

```diff
@@ -9,7 +9,7 @@ use std::{
 use bytes::{Bytes, BytesMut};
 use futures::{ready, stream::Stream};
 use pin_project::unsafe_project;
```

Comment:
Travis freaks out on this line, `use pin_project::unsafe_project;` should be below libzstd